loadgen/README.md

```diff
@@ -103,3 +99,67 @@ with the reference models.
 
 For templates of how to do the above in detail, refer to code for the demos,
 tests, and reference models.
+
+
+## Submissions over the network
+
+For reference, on a high level a submission looks like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_submission.png" width="300px" style="padding: 20px">
+</div>
+
+The loadgen implementation is common to all submissions, while the QSL (“Query Sample Library”) and SUT (“System Under Test”) are implemented by submitters. QSL is responsible for loading the data and includes untimed preprocessing.
+
+A submission over the network introduces a new component “QDL” (query dispatch library) and a it would look like this:
```

Comment:
"and it would look this"

Reply:
Suggest to change to:
"A submission over the network introduces a new component “QDL” (query dispatch library) that is added to the system as presented in the following diagram:"